sphinx/source/docs/user_guide/server.rst

```diff
@@ -1285,62 +1262,64 @@ It must contain either:
     def get_login_url(request_handler):
         pass
 
-When a relative ``login_url`` is given, an optional ``LoginHandler`` class may
-also be provided, and it will be installed as a route on the Bokeh server
+If the module provides a relative ``login_url``, it can also provide an
+optional ``LoginHandler`` class, which the Bokeh server will incorporate
 automatically.
 
 The ``get_login_url`` function is useful in cases where the login URL must
-vary based on the request, or cookies, etc. It is not possible to specify a
-``LoginHandler`` when ``get_url_function`` is defined.
+vary based on the request, cookies, or other factors. You can also specify a
+``LoginHandler`` when defining the ``get_url_function``.
 
-Analogous to the login options, optional ``logout_url`` and ``LogoutHandler``
-values may be used to define an endpoint for logging users out.
+To define an endpoint for logging users out, you can also use optional
+``logout_url`` and ``LogoutHandler`` parameters, similar to the login options.
 
-If no auth module is provided, a default user will be assumed, and no
-authentication will be required to access Bokeh server endpoints.
+If you don't provide an authentication module, the configuration will not
+require any authentication to access Bokeh server endpoints.
 
 .. warning::
-    The contents of the auth module will be executed!
+    The configuration executes the contents of the authentication module.
 
-Secure Cookies
+Secure cookies
 ''''''''''''''
 
 If you want to use Tornado's `set_secure_cookie`_ and `get_secure_cookie`_
-functions in your auth module, a cookie secret must be set. This can be
-accomplished with the ``BOKEH_COOKIE_SECRET`` environment variable. e.g.
+functions in your auth module, you'll have to set a cookie secret. To do so,
+use the ``BOKEH_COOKIE_SECRET`` environment variable.
 
 .. code-block:: sh
 
     export BOKEH_COOKIE_SECRET=<cookie secret value>
 
-The value should be a long, random sequence of bytes
+The value should be a long, random sequence of bytes.
 
 .. _userguide_server_deployment_security:
 
 Security
 ~~~~~~~~
 
-By default, a Bokeh server will accept any incoming connections on allowed
-websocket origins. If a session ID is specified, and a session with that ID
-already exists on the server, then a connection to that session is made.
-Otherwise, a new session is automatically created and used.
+By default, the Bokeh server will accept any incoming connections with an
```

Comment:
cc @tcmetzger for thoughts

I had used "a Bokeh server" here to emphasize "a concrete, actually running Bokeh server process" as opposed to what (to me) reads more like an abstract concept of "the Bokeh server" but maybe I am being pedantic.

Reply:
I'd be ok with "the Bokeh Server". We are talking about the server's default behavior, and allowing those connections seems to be true for any instance of a Bokeh server (in its default configuration) - this could therefore be considered a property of the abstract concept of "the Bokeh server".